image/cli/mascli/templates/pipelinerun.yaml

```diff
@@ -175,6 +190,12 @@ spec:
       value: '$MAS_CATALOG_SOURCE'
     - name: mas_catalog_version
       value: '$MAS_CATALOG_VERSION'
+    - name: mas_add_channel
+      value: '$MAS_ADD_CHANNEL'
+    - name: mas_add_catalog
```

Comment:
these mas_add_* are also added lower day rather than here

Reply:
Removed the dups from `/templates/pipelinerun.yml`